src/System.Net.Security/src/System/Net/SecureProtocols/_FixedSizeReader.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Net
+{
+    //
+    // Reads a fixed size packet from a stream, can disover EOF as 0 bytes read from stream.
+    //
+    internal class FixedSizeReader
+    {
```

Comment:
I agree. 41 files have this naming in our staging branch across multiple namespaces. Tracking with #3116